packs/salt/actions/lib/utils.py

```diff
@@ -0,0 +1,30 @@
+import yaml
+
+action_meta = {
+    "name": "",
+    "parameters": {
+        "action": {
+            "type": "string",
+            "immutable": True,
+            "default": ""
+        },
+        "kwargs": {
```

Comment:
@manasdk This too is from st2incubator. 

This pack closely mirrors to how actions are handed in the [aws pack](https://github.com/StackStorm/st2contrib/tree/master/packs/aws/actions). The approach allows for the usage of a single entry point while using the yaml files themselves as a way to namespace the registered actions.